src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1141,6 +1176,22 @@ export class AmpStoryPlayer {
       });
   }
 
+  /**
+   * @param {!StoryDef} story
+   * @param {!Element} iframe
+   * @param {!VisibilityState} visibilityState
+   * @return {!Promise}
+   * @private
+   */
+  layoutIframe_(story, iframe, visibilityState) {
```

Comment:
This extra layer of abstraction seems unnecessary, should all other calls to layoutIframe just call `updateIframeSrc_` now?

Reply:
Yeah just didn't want to change more code :p I updated it in the latest commit. PTAL.